app/src/androidTest/java/ch/epfl/favo/view/FavorDetailViewTest.java

```diff
@@ -184,18 +184,10 @@ public void testAcceptFlow() {
   @Test
   public void testCancelFlowByAccepter() throws Throwable {
     // favor is cancelled by accepter
-    onView(withId(R.id.accept_button)).perform(click());
     getInstrumentation().waitForIdleSync();
-    onView(withId(R.id.accept_button))
-        .check(matches(withText(R.string.cancel_accept_button_display)))
-        .perform(click());
+    onView(withId(R.id.accept_button)).perform(click());
     getInstrumentation().waitForIdleSync();
 
-    // check snackbar shows
-    onView(withId(com.google.android.material.R.id.snackbar_text))
-        .check(matches(withText(R.string.favor_cancel_success_msg)));
-    checkCancelledView(FavorStatus.CANCELLED_ACCEPTER);

```

Comment:
Not a big issue, but these codes are used to check the view when a favor is cancelled by accepter, texts at app bar are different compared to the case cancelled by requester, in terms of FavorDetail View.

Reply:
Yeah you're right, I put it back as it was before. I modified it because the test passes on Cirrus but not locally for me. However, I think the problem is related to the layout and Espresso can't find some buttons on some emulators (I think David had the same problem) because of the way the elements are arranged. Probably implementing a scroll view and a simpler UI will fix these problems but we should keep an eye on this.